client/subcomponents/dataVis/BarGraph.jsx

```diff
@@ -6,6 +6,7 @@ import { Group } from '@visx/group'
 // import { ScaleSVG } from '@visx/responsive';
 import { scaleBand, scaleLinear } from '@visx/scale'
 import { AxisBottom, AxisLeft } from '@visx/axis'
+// import { Grid, LineSeries, XYChart, Axis, Tooltip } from '@visx/xychart'
```

Comment:
please remove this and any other commented out code. granted, this doc has a lot of useful comments in it, but if there is any commented out code or comments that are not essential please remove them :)